sites/geohub/src/components/controls/VectorLegendAdvanced.svelte

```diff
@@ -40,38 +40,38 @@
   import { getMaxValueOfCharsInIntervals } from '$lib/helper/getMaxValueOfCharsInIntervals'
   import { updateIntervalValues } from '$lib/helper/updateIntervalValues'
 
-  export let applyToOption: VectorApplyToTypes = VectorApplyToTypes.COLOR
+  export let applyToOption: VectorApplyToTypes
   export let layer: Layer
   export let layerMax: number
   export let layerMin: number
   export let colorMapName: string
   export let defaultColor: string
+  export let propertySelectValue
+  export let numberOfClasses: number
+  export let colorMapRows: IntervalLegendColorMapRow[]
+  export let defaultOutlineColor: string
```

Comment:
these four properties may not need to be exported. and all  information can be extracted from layer style object